src/tasks/siri-vm-downloader/index.ts

```diff
@@ -0,0 +1,28 @@
+import { getDatabaseClient } from "@bods-integrated-data/shared/database";
+import { fastifySensible } from "@fastify/sensible";
+import Fastify from "fastify";
+import Pino from "pino";
+import healthCheck from "./routes/health";
+import downloadSiriVm from "./routes/siri-vm";
+
+const fastify = Fastify({
+    logger: Pino(),
```

Comment:
Is there a specific reason for adding Pino() in like this? Looking at the docs, it looks like Fastify already uses Pino and all we need to pass it it logger: true to enable logging.